File: myJava_01/Exception.java
import java.util.Scanner;
public class Exception {
    public static void main(String args[])
    {
        Scanenr sc = new Scanner(System.in);

        System.out.println("This program result the division"); 
        
        int a = sc.nextInt();
        int b = sc.nextInt();
        try {
            if(b==0) throw 1;
        } catch (int x) {
            System.out.println("Cannot divided by zero!");
        }

        System.out.println((double)a/(double)b);
    }
}

File: myJava_01/Exception.cpp
#include <iostream>

int main ()
{
    
    int a;
    int b;


    std::cout << "This result the division" << std::endl;

    std::cout << "input a:";
    std::cin >> a;

    std::cout << "input b:";
    std::cin >> b;
    
    try
    {
        
        if(b == 0)
        {
            throw 1;
        }

        std::cout<< "Result: ";
        std::cout << (double)a/(double)b << std::endl;
    }
    catch (...)
    {
        std::cout << "Can't divided by 0" << std::endl;
    }

    return 0;

}